eng/pipelines/coreclr/templates/run-scenarios-job.yml

```diff
@@ -127,7 +131,12 @@ jobs:
       displayName: Build Startup tool (Linux)
       env:
         PERFLAB_TARGET_FRAMEWORKS: net6.0
-      condition: and(succeeded(), ne(variables['Agent.Os'], 'Windows_NT'))
+      condition: and(succeeded(), and(ne(variables['Agent.Os'], 'Windows_NT'), ne(variables['Agent.Os'], 'Darwin')))
```

Comment:
I don't fully remember if the `Agent.Os` string is consistent for Linux, but if it is we should look into changing these conditions from negative to positive, since we now have a third OS to choose from.

Reply:
Per https://docs.microsoft.com/en-us/azure/devops/pipelines/build/variables?view=azure-devops&tabs=yaml#agent-variables-devops-services, it looks like the three Agent.OSs are Windows_NT, Darwin, and Linux